selfdrive/test/process_replay/process_replay.py

```diff
@@ -336,12 +338,34 @@ def ublox_rcv_callback(msg):
   ),
 ]
 
+def setup_prefix():
+  os.environ['OPENPILOT_PREFIX'] = str(uuid.uuid4())
+  msg_path = '/dev/shm/' + os.environ['OPENPILOT_PREFIX']
+  try:
+    os.mkdir(msg_path)
+  except FileExistsError:
+    pass
+  except PermissionError:
+    print(f"WARNING: failed to make {msg_path}")
+
+
+def teardown_prefix():
+  params_path = '/data/params/' if os.environ.get('TICI', 0) else os.environ['HOME'] + '/.comma/params/'
```

Comment:
should be able to get this from params directly. Params already has a getParamPath you can use, but it just needs to be exposed in the cython wrapper

Reply:
We have common/basedir.py which handles this for persist. Could be nice to use it for params aswell.